pyanaconda/modules/common/task.py

```diff
@@ -26,11 +26,24 @@
 
 from pyanaconda.core.signal import Signal
 from pyanaconda.core.constants import THREAD_DBUS_TASK
-from pyanaconda.task.task_interface import TaskAlreadyRunningException
+from pyanaconda.modules.common.task_interface import TaskInterface
+from pyanaconda.modules.common.task_interface import TaskAlreadyRunningException
```

Comment:
I'm not fan of having everything in one folder. What about having `pyanaconda.modules.common.task.interface` instead of what is there now.

You know Tasks don't have anything related to base module classes so they shouldn't be together.

Reply:
Moved to a new package pyanaconda.common.task.